### PR TITLE
fix: [#163] added part to svg so users can also resize the six logo spinner

### LINKED ISSUE #163: Sizing SIX Logo Spinner
### What problem does this feature solve?

could you add sizing parameter like for the normal spinner also to the logo spinner. 
https://six-group.github.io/six-webcomponents/components/six-spinner.html#size

### What does the proposed API look like?

What is working right now is the following spinner:
  <six-spinner style="font-size: 10rem"></six-spinner>
Now it would be nice if its possible to adjust the size the same way on the logo spinner:
  <six-spinner style="font-size: 10rem" six="true"></six-spinner>

### PR DESCRIPTION
**Pull request by Riatomerehentroweard**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Resolves #163 

Introduced part attribute to svg in six-spinner so user can style the six-logo styled spinner.

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] It's submitted to the `main` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] I have updated the documentation accordingly.
- [ ] All tests are passing
- [ ] New/updated tests are included
- [ ] I have updated the "upcoming" section inside _docs/changelog.md_ explaining the changes I contributed

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: docs/components/six-spinner.md [hunk 1]
```diff
@@ -74,6 +100,7 @@ Spinner can be configured as animated SIX logo.
 | Part     | Description                   |
 | -------- | ----------------------------- |
 | `"base"` | The component's base wrapper. |
+| `"svg"`  |                               |
```
Comment (pennal) [inline docs/components/six-spinner.md:103]: Add a description, even if self explanatory

File: libraries/ui-library/src/components/six-spinner/readme.md [hunk 2]
```diff
@@ -17,6 +17,7 @@
 | Part     | Description                   |
 | -------- | ----------------------------- |
 | `"base"` | The component's base wrapper. |
+| `"svg"`  |                               |
```
Comment (pennal) [inline libraries/ui-library/src/components/six-spinner/readme.md:20]: same here

File: docs/components/six-spinner.md (hunk 1 above)
Comment (Riatomerehentroweard) [reply docs/components/six-spinner.md]: done

**Review by davidemarcoli — APPROVED** [review]

File: libraries/ui-library/src/components/six-spinner/readme.md (hunk 2 above)
Comment (Riatomerehentroweard) [reply libraries/ui-library/src/components/six-spinner/readme.md]: done

**Pull request merged**